src/main/java/codeu/enumeration/ActivityTypeEnum.java

```diff
@@ -0,0 +1,7 @@
+package codeu.enumeration;
+
```

Comment:
I'd rather put this with the Activity class. I'm guessing this is used in conjunction with the Activity class.

Reply:
It can be used outside of the Activity class as well, when passing values to the constructor. If I understand correctly, putting it as an inner Enum of the Activity model makes it only accessible by the Activity class. And, although more of a choice of design, I prefer for Models to be a pure representation of their corresponding entities